cpu/efm32/cpu.c

```diff
@@ -124,17 +124,17 @@ static void clk_init(void)
     /* set (and enable) the LFB clock source */
     CMU_ClockSelectSet(cmuClock_LFB, CLOCK_LFB);
 
-#ifdef _SILICON_LABS_32B_SERIES_1
+#if defined(_SILICON_LABS_32B_SERIES_1)
     /* set (and enable) the LFE clock source */
     CMU_ClockSelectSet(cmuClock_LFE, CLOCK_LFE);
 #endif
 
     /* disable the LFRCO if external crystal is used */
     if (CLOCK_LFA == cmuSelect_LFXO && CLOCK_LFB == cmuSelect_LFXO &&
-#ifdef _SILICON_LABS_32B_SERIES_1
-        CLOCK_LFE == cmuSelect_LFXO)
-#else
+#if defined(_SILICON_LABS_32B_SERIES_0)
         true)
+#elif defined(_SILICON_LABS_32B_SERIES_1)
+        CLOCK_LFE == cmuSelect_LFXO)
 #endif
```

Comment:
```C
#if defined(_SILICON_LABS_32B_SERIES_0)
#define LFE_IS_LFXO (true)
#elif defined(_SILICON_LABS_32B_SERIES_1)
#define LFE_IS_LFXO (CLOCK_LFE == cmuSelect_LFXO)
#endif
```

```suggestion
        LFE_IS_LFXO)
```

Reply:
I already know that I will refactor this when adding support for Series 2, so I want to keep it as-is, at this moment.

Series 2 doesn't have a LFE clock branch.

Reply:
Travis doesn't like it though

```
cpu/efm32/cpu.c:133: error (syntaxError): Invalid number of character '(' when no macros are defined.
cpu/efm32/cpu.c:133: error (syntaxError): Invalid number of character '(' when these macros are defined: 'DCDC_PRESENT'.
cpu/efm32/cpu.c:133: error (syntaxError): Invalid number of character '(' when these macros are defined: 'DCDC_PRESENT;EMU_DCDCINIT_OFF'.
```

Reply:
I made the `#else`-statement explicit, so that it always defines a proper `if`-statement.